our-brothers-web/src/app/components/drop-down-menu/drop-down-menu.component.ts

```diff
@@ -8,6 +8,8 @@ import { Component, OnInit, Input, TemplateRef } from '@angular/core';
 export class DropDownMenuComponent implements OnInit {
   @Input() menuContent: TemplateRef<any>;
   @Input() menuAnchor: TemplateRef<any>;
+  @Input() disabled: boolean = false;
+  @Output() click = new EventEmitter<any>();
```

Comment:
Please remove

Reply:
this helps disable the drop down when we drop below a certain size

Reply:
Cool, I meant the `click` event

Reply:
removed